src/Pos/Util/UserSecret.hs

```diff
@@ -109,6 +158,7 @@ peekUserSecret path = takeReadLock path $ do
 -- 'writeUserSecretRelease'.
 takeUserSecret :: (MonadIO m) => FilePath -> m UserSecret
 takeUserSecret path = liftIO $ do
+    initializeUserSecret path
```

Comment:
@vincenthz AFAIU in the current implementation it's a user directory (for me, the `*.key` files are created in the root of the cloned `cardano-sl` repo).